lib/plugins/villager.js

```diff
@@ -152,31 +155,12 @@ function inject (bot, { version }) {
       if (Trade.hasItem2) proms.push(once(villager, 'updateSlot:1'))
       if (bot.supportFeature('setSlotAsTransaction')) {
         proms.push(once(villager, 'updateSlot:2'))
-        await new Promise((resolve, reject) => {
-          let countOfItemOneLeftToTake = itemCount1 > 64 ? 64 : itemCount1
-          let countOfItemTwoLeftToTake = 0
-          if (Trade.hasItem2) {
-            countOfItemTwoLeftToTake = itemCount2 > 64 ? 64 : itemCount2
-          }
-          const listener = (slot, oldItem, newItem) => {
-            if (!(slot >= villager.inventoryStart && slot <= villager.inventoryEnd)) return
-            if (newItem === null) {
-              if (oldItem.type === Trade.inputItem1.type) countOfItemOneLeftToTake -= oldItem.count
-              else if (Trade.hasItem2 && oldItem.type === Trade.inputItem2.type) countOfItemTwoLeftToTake -= oldItem.count
-            }
-            if (countOfItemOneLeftToTake === 0 && countOfItemTwoLeftToTake === 0) {
-              villager.off('updateSlot', listener)
-              resolve()
-            }
-          }
-          villager.on('updateSlot', listener)
-        })
```

Comment:
The bot consistently gets hung up here, and I honestly can't figure out what the purpose of this is. It's possible that it was necessary before some of the other changes I made. @nickelpro do you remember by chance?